sopel/bot.py

```diff
@@ -714,11 +714,15 @@ def call(
         """Call a function, applying any rate limits or other restrictions.
 
         :param func: the function to call
-        :type func: :term:`function`
```

Comment:
Removing this is an interesting choice when the type annotation still says `Any`

Reply:
Yeah bad idea.